scripts/configure-hypervisor.sh

```diff
@@ -226,7 +226,13 @@ generate_qemu_options() {
 	qemu_options+=(size:--disable-glusterfs)
 	qemu_options+=(size:--disable-libiscsi)
 	qemu_options+=(size:--disable-libnfs)
-	qemu_options+=(size:--disable-libssh2)
+
+	# Starting from QEMU 4.1, libssh replaces to libssh2
+	if [ "${qemu_version_major}${qemu_version_minor}" -ge 41  ]; then
```

Comment:
canny trick... ;-)

Reply:
hmm, but I guess that fails if we ever get qemu v3.10 ;-)

Reply:
@grahamwhaley ahh - you're right,

Reply:
if you rework, maybe some bash internal 'printf %02x' type 'fixed width' printing might be a way to workaround, and still use the nice numeric compare trick ;-)

Reply:
If you are just using major + minor, you can use `bc(1)` for version comparisons.